v2/internal/controllers/crd_machinelearningservices_20240401_test.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package controllers_test
```

Comment:
I think we've been naming these files following the new naming pattern which is <service>_<resoruce>_<version>_test.go - so in this case `machinelearning_workspaces_20240401_test.go`.

I don't think changing this file name impacts the test recordings at all so it should be easy to change.